src/gmt2kml.c

```diff
@@ -313,7 +313,7 @@ static int parse (struct GMT_CTRL *GMT, struct GMT2KML_CTRL *Ctrl, struct GMT_OP
 
 		switch (opt->option) {
 
-			case '<':	/* Input files */
+			case '<':	/* Input files (we only check one) */
 				if (n_files++ > 0) break;
```

Comment:
Should this line be updated?

Reply:
Thanks, yes, now fixed.
Also found lots of ;; when there should just be one.